subgraph.template.yaml

```diff
@@ -33,11 +77,16 @@ dataSources:
         - FixedEarningsUpdate
         - AccumulatorAccrual
         - FloatingDebtUpdate
+        - Account
+        - FixedPosition
+        - MaxFuturePoolsSet
+        - PenaltyRateSet
+        - ReserveFactorSet
+        - BackupFeeRateSet
+        - MarketState
       abis:
-        - name: {{ name }}
+        - name: Market
```

Comment:
`ERC20` ABI is missing for Market. Why this change?

Reply:
not specifically the change of the diff above, I mean the changes of [🚚 refactor generated modules](https://github.com/exactly/subgraph/pull/3/commits/5e2cfa5aae658f46425df18070e38a5520849433)